dubbo-metadata-report-extensions/dubbo-metadata-report-etcd/src/main/java/org/apache/dubbo/metadata/store/etcd/EtcdMetadataReport.java

```diff
@@ -126,6 +126,16 @@ public String getServiceDefinition(MetadataIdentifier metadataIdentifier) {
         return etcdClient.getKVValue(getNodeKey(metadataIdentifier));
     }
 
+    @Override
+    public boolean registerServiceAppMapping(String key, String group, String content, Object ticket) {
+        String appKey = toRootDir() + "mapping-data" + PATH_SEPARATOR + key + PATH_SEPARATOR + group;
```

Comment:
why use mapping-data ?

Reply:
> why use mapping-data ?

The "mapping-data" string is used as a part of the key for storing service application mappings in Etcd.  In the context of this code, a service application mapping is a relationship between a service and an application, which is stored in Etcd for later retrieval.

Reply:
<img width="922" alt="image" src="https://github.com/apache/dubbo-spi-extensions/assets/81170548/970cfd4a-ce47-42e2-bc89-baf6795f0849">
i found it has contanis mapping keywords

Reply:
![image](https://github.com/apache/dubbo-spi-extensions/assets/7471716/6d19c044-5c0c-4824-937e-8d90bda40c5a)
That's okay, I just want metadata to look the same as mapping-data